### ISSUE TITLE
Not nullable queries

### ISSUE THREAD

**Issue by lishine** [danielrearden/sqlmancer#49]
```
    type Query @sqlmancer(dialect: POSTGRES, transformFieldNames: SNAKE_CASE) {
        hello(name: String): String!
        isAuth: AuthPayload!
        orders: Order! @many @paginate
    }
```
Here I set orders not nullable - Order!
But the result is orders: OrdersPage. not - orders: OrdersPage!
Is it fine?
I mean, if I would like to use graphql not nullable queries, here it will not work...
Or you would not recommend using not nullable queries?

**Comment by danielrearden** [comment]
Just a bug -- `OrdersPage` should always be non-null regardless of the initial type.

**Comment by danielrearden** [comment]
Fixed with 65e4bfd9a201e82e84397a701846b27c20d06857
